pkg/compiler/compiler.go

```diff
@@ -173,7 +182,7 @@ func CompileAndSave(src string, o *Options) ([]byte, error) {
 	if len(o.Ext) == 0 {
 		o.Ext = fileExt
 	}
-	b, di, err := CompileWithDebugInfo(src, nil)
+	b, di, err := CompileWithOptions(src, nil, o)
```

Comment:
Maybe we can add a separate flag to the `contract compile` command to be able to use this option from the outside? Like it is done for `NoStandardCheck` and `NoEventsCheck`.

Reply:
Oh, I see that `no-events` is used for that.